Firestore/Example/Tests/Integration/FSTTransactionTests.mm

```diff
@@ -28,7 +28,7 @@ @implementation FSTTransactionTests
 
 // We currently require every document read to also be written.
 // TODO(b/34879758): Re-enable this test once we fix it.
-- (void)xtestGetDocuments {
+- (void)testGetDocuments {
```

Comment:
I'd recommend removing the TODO on the line above as it no longer makes sense (there's nothing to re-enable). There's already a TODO for this issue within the body (line 46)

Reply:
done